### DISCUSSION TITLE
1.4.0-beta.0

### DISCUSSION THREAD

**Discussion by louislam** [louislam/dockge#323] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/louislam/dockge/releases/tag/1.4.0-beta.0'>1.4.0-beta.0</a>.</em>

**Comment by Procuria** [comment]
Thank you for this awesome Christmas present 🤓

**Comment by iball** [comment]
Works like a charm for me after jumping my 4 dockge containers over to the beta.  THANKS!
Would be nice if we could name/rename the agents' name in the GUI though, but I'm sure you're already working on that.

**Reply by louislam** [reply]
Thanks for testing it!

**Comment by JordyEGNL** [comment]
I have just set this up on my servers and it is working great!!

Would like to see an option to have the docker containers grouped by their server :)

**Comment by aatumaykin** [comment]
Cool 🔥
<img width="281" alt="image" src="https://github.com/louislam/dockge/assets/704282/bd078f56-b70a-4f87-a2d7-c6fbecfcc8be">
